### PR TITLE
Router should not prevent default if meta keys are held down

### PR DESCRIPTION
**Pull request by baile320**
This PR should close https://github.com/sycamore-rs/sycamore/issues/129.

I'm not 100% sure of the logic, because in the SvelteKit router code that was referenced in the issue (https://github.com/sveltejs/kit/blob/8854e2ffc48cf653854828f6ac19b2ce620535a6/packages/kit/src/runtime/client/router.js#L103), the key presses were checked immediately and if found the function immediately returned. So, in the code below, I skipped all of the subsequent logic if any of those keys were pressed. I'm not sure if the behavior is that we expect only the ```prevent_default``` from happening (in which case, I can update those two sections separately).

### CODE REVIEW

File: packages/sycamore-router/Cargo.toml [hunk 1]
```diff
@@ -27,5 +27,6 @@ features = [
   "Location",
   "PopStateEvent",
   "Window",
+  "KeyboardEvent",
```
Comment (lukechu10) [inline packages/sycamore-router/Cargo.toml:30]: Nit: could you please specify the features in alphabetical order

**Review by lukechu10 — APPROVED** [review]
lgtm.
We don't want `prevent_default` from running because that would override the browser's built-in behavior when these keys are pressed.

File: packages/sycamore-router/Cargo.toml (hunk 1 above)
Comment (baile320) [reply packages/sycamore-router/Cargo.toml]: done!

**Comment by lukechu10** [comment]
Thanks for contributing!

**Pull request merged**